lib/client-assets.php

```diff
@@ -664,6 +664,15 @@ function gutenberg_extend_block_editor_styles( $settings ) {
 		array_unshift( $settings['styles'], $editor_styles );
 	}
 
+	// Remove the default font editor styles.
+	// When Gutenberg is updated to have minimum version of WordPress 5.8
+	// This could be removed.
+	foreach ( $settings['styles'] as $j => $style ) {
+		if ( 0 === strpos( $style['css'], "body { font-family:" ) ) {
+			unset( $settings['styles'][ $j ] );
+		}
+	}
```

Comment:
Core automatically loads a style setting a default font family, this removes that editor style (reset to browser defaults)